architecture/core_concepts/overview.adoc

```diff
@@ -7,10 +7,8 @@
 
 The following topics provide high-level, architectural information on core
 concepts and objects you will encounter when using OpenShift. Many of these
-objects come from link:../../rest_api/kubernetes_v1.html[the model provided by
-Kubernetes], which is link:../../rest_api/openshift_v1.html[extended by
-OpenShift's model] to provide a more feature-rich development lifecycle
-platform.
```

Comment:
Hey @sosiouxme -- It's just not the type of content I think we should be linking to from an overview. As you said, it's not very overview-y. I reckon if someone wants to check out the nuts & bolts, the API content is pretty top-level and easy to find in the docs. :cactus: